js/start.js

```diff
@@ -394,6 +425,18 @@ app.apiSocket.on('message', (e) => {
 
 let unpublishedConfirm;
 
+app.apiSocket.on('message', (e) => {
+  if (e.jsonData) {
+    if (e.jsonData.notification) {
+      if (e.jsonData.notification.follow) {
+        app.ownFollowers.add({ guid: e.jsonData.notification.follow });
```

Comment:
When someone is followed, in the DB, they are added to the top off the list. I would think we want to mimic that. I think Collection.add() will add the model to the end of the list, so we probably want to utilize Collection.unshift() here.